io.github.ceprogramming.CEmu.metainfo.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.ceprogramming.CEmu</id>
+	<launchable type="desktop-id">io.github.ceprogramming.CEmu.desktop</launchable>
+	<metadata_license>CC0-1.0</metadata_license>
+	<name>CEmu</name>
+	<summary>A third-party TI-84 Plus CE emulator</summary>
+	<description>
+		<p>CEmu is a third-party TI-84 Plus CE / TI-83 Premium CE calculator emulator, focused on developer features. CEmu works natively on Windows, macOS, and Linux. For performance and portability, the core is programmed in C and its customizable GUI in C++ with Qt.</p>
+	</description>
+	<categories>
+		<category>Development</category>
+	</categories>
+	<url type="homepage">https://ce-programming.github.io/CEmu/</url>
+	<url type="bugtracker">https://github.com/CE-Programming/CEmu/issues</url>
+	<project_license>GPL-3.0-only</project_license>
+	<developer_name>The CEmu Developers</developer_name>
+	<screenshots>
+		<screenshot type="default">
+	    		<image type="source" width="2583" height="1488">https://imgur.com/7QsJd5L</image>
```

Comment:
imgur is never appropriate to host screenshot.

Also this URL return a code 429.